### DISCUSSION TITLE
Add powered launch with passing station to LSM launched coaster

### DISCUSSION THREAD

**Discussion by ZeeMaji** [OpenRCT2/OpenRCT2#23124] [Feature requests]
Exactly what it says, the coaster currently has only the without passing station mode available. Both modes should be available as every other coaster with the launched modes has both with and without passing station types available. Plus i think it has creative potential for creating swing launch layouts and it should not affect balance as the without passing station mode the ride type already has is always better than with passing station.

**Comment by X123M3-256** [comment]


**Reply by ZeeMaji** [reply]
Imo i'd just adhere to the standard of having both modes available just for the sake of consistency with all other ride types that have powered launches.
